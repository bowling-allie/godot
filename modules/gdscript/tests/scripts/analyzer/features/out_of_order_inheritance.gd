func test():
	print("test")
	var t = preload("out_of_order_inheritance_a.notest.gd").new()
	print(t.test_val)
