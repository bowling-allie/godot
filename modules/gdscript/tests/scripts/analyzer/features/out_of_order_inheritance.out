GDTEST_OK
waiting for godot
